rofication/_dbus.py

```diff
@@ -81,9 +92,18 @@ def Notify(self, app_name: str, replaces_id: int, app_icon: str, summary: str,
 class RoficationDbusService:
     def __init__(self, queue: NotificationQueue) -> None:
         # preserve D-Bus object reference
+        self.queue = queue
         self._object = RoficationDbusObject(queue)
         # create GLib mainloop, this is needed to make D-Bus work and takes care of catching signals.
         self._loop = MainLoop()
 
     def run(self) -> None:
+        timeout_add_seconds(60, self.backup_queue)
```

Comment:
I would prefer it if this behavior is opt-in.  It's a matter of opinion either way but I reckon that most users would not want this functionality as it comes at the cost of periodic disk writes.

Reply:
I disagree with those odds :-) I definitely prefer some I/O to notification loss. Note that the write only happens if there are unsaved notifications.
Anyway, I'm made it configurable with default to no save. I know that I will keep my notifications safe, but well :-)